include/battle.h

```diff
@@ -718,6 +718,7 @@ struct BattleStruct
     bool8 trainerSlideBeforeFirstTurnMsgDone;
     u32 aiDelayTimer; // Counts number of frames AI takes to choose an action.
     u32 aiDelayFrames; // Number of frames it took to choose an action.
+    bool8 transformZeroToHero[PARTY_SIZE];
```

Comment:
I think we need to start looking into using a struct for tracking party-specific stuff, because using a whole byte for each mon is a bit uneeded. Also, this doesn't look like it's tracking both sides correctly.

Reply:
The second part should be fixed.